JASP-Tests/R/tests/testthat/test-networkanalysis.R

```diff
@@ -0,0 +1,43 @@
+context("Network Analysis")
+
+# does not test
+# - error handling
+
+test_that("All tables results match", {
+  options <- JASPTools::analysisOptions("NetworkAnalysis")
+  options$variables <- c("contNormal", "contcor1", "contcor2")
+  options$tableCentrality <- TRUE
+  options$tableClustering <- TRUE
+  options$tableWeightsMatrix <- TRUE
+  options$tableLayout <- TRUE
+  results <- JASPTools::run("NetworkAnalysis", "debug.csv", options, view=FALSE, quiet=TRUE, , sideEffects = "pkgLoading")
+
+  table <- results[["results"]][["generalTB"]][["data"]]
+  expect_equal_tables(table,
+    list("Network", 3, "2 / 3", 0.333333333333333)
```

Comment:
Example of an added label: `list("Network", 3, "2 / 3", 0.333333333333333), label="generalTB"`